lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup_facts.py

```diff
@@ -193,6 +240,8 @@ def main():
         show_network_policy=dict(type='bool', default=True),
         show_teaming_policy=dict(type='bool', default=True),
         show_port_policy=dict(type='bool', default=True),
+        dvswitch=dict(),
```

Comment:
could you explicitly define the format of the `dvswitch` parameter: `dvswitch=dict(type='str', required=False)`

Reply:
@goneri I think these are defaults so not required.

Reply:
Waiting for final answer.